### ISSUE TITLE
Wondering if I need to worry about this ARIA thing

### ISSUE THREAD

**Issue by fredriksundstrom** [reactjs/react-autocomplete#215]
Hi! Nice component, and says it’s ARIA compliant. I tried to use it, and using one of the examples, I get these results when running it through Lighthouse. Just wanted to know if it's something I need to fix/worry about?:

1.
Elements with ARIA roles have the required aria-* attributes   
Fail
1 element failed this test
#states-autocomplete

2.
Every form element has a label   
Fail
1 element failed this test
#states-autocomplete

Thanks!

**Comment by CMTegner** [comment]
It's hard to tell without having the source which you ran through lighthouse, but number 2 is probably because you didn't include a `<label>` element with a corresponding `htmlFor` attribute. `Autocomplete` doesn't do this for you, it only provides the `<input>` element.

I'd be happy to look at the first failure if you can provide more context.

**Comment by fredriksundstrom** [comment]
Thank you @CMTegner ! I’m not sure what context you’d want to look at, but if you give me an idea I’d be happy to paste code or show you what’s needed! It’s currently live here: https://healthgoing-stage.herokuapp.com/

You were absolutely right about #2, of course.

**Comment by CMTegner** [comment]
Looks like `aria-expanded` is missing. I pushed 1.4.1 which adds the missing attribute. Let me know if you experience any more problems.

**Comment by fredriksundstrom** [comment]
My test passes so it’s working out fine, thank you @CMTegner ! 👍
